assets/js/modules/forms.js

```diff
@@ -44,6 +44,18 @@ function toggleReviewAnswers() {
     });
 }
 
+function handleExpandingDetails() {
+    let isClosed = true;
+    const $toggleBtn = $('.js-toggle-all-details');
+    $toggleBtn.text($toggleBtn.data('label-closed')).show();
+
+    $toggleBtn.on('click', function() {
+        $toggleBtn.text(isClosed ? $toggleBtn.data('label-open') : $toggleBtn.data('label-closed'));
+        $('details').attr('open', isClosed);
```

Comment:
I'd be tempted to scope this to a specific class, or parent element. _All_ details on the page might get a little weird. Might be overthinking it but we don't know where else we might want to use details on this page.